### PR TITLE
Update deps. Look up pods from all namespaces.

### LINKED ISSUE #31: Namespaced containers stats aren't reported
Not sure if it's a problem with cAdvisor, Heapster or Kubernetes but using [kubernetes setup for monitoring](https://github.com/GoogleCloudPlatform/kubernetes/tree/master/examples/monitoring) I don't see in grafana pods started in namespaces other than the default one.

### PR DESCRIPTION
**Pull request by vishh**
Fixes #31

### CODE REVIEW

**Comment by vishh** [comment]
cc @rjnagal

File: Godeps/Godeps.json [hunk 1]
```diff
@@ -143,18 +151,18 @@
 		},
 		{
 			"ImportPath": "github.com/google/cadvisor/client",
-			"Comment": "0.5.0-76-gd74485c",
-			"Rev": "d74485ce77d81101c059efc0841deb44ae177258"
+			"Comment": "0.5.0-165-g8064a13",
```
Comment (rjnagal) [inline Godeps/Godeps.json]: Do we want to move it to 0.7.1?

Comment (vishh) [inline Godeps/Godeps.json]: Update deps. I don't think the version number here matches the cadvisor's versioning system.

**Comment by vishh** [comment]
PTAL @rjnagal

**Comment by rjnagal** [comment]
LGTM

**Pull request merged**